src/main/kotlin/com/github/vol0n/utbotcppclion/server/LogInterceptor.kt

```diff
@@ -0,0 +1,43 @@
+package com.github.vol0n.utbotcppclion.server
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import testsgen.Testgen
+
+class LogInterceptor : io.grpc.ServerInterceptor {
```

Comment:
This class intercepts incoming messages and can be used for logging them

Reply:
When does it intercept messages? Could you please describe the case when you need this class in comments

Reply:
It intercepts every message, so it can be seen what messages are delivered and when, what was the last message. Usecase: you can see the message's fields, for example what paths are sent, it is useful for debugging